src/test/java/it/pn/frontend/e2e/stepDefinitions/common/HelpdeskTest.java

```diff
@@ -20,18 +22,36 @@ public class HelpdeskTest {
 
     private final Logger logger = LoggerFactory.getLogger("HelpdeskAppTest");
 
-    HelpdeskPage helpdeskPage = new HelpdeskPage(this.driver);
+    private HelpdeskPage helpdeskPage = new HelpdeskPage(this.driver);
     private Map<String, Object> datiTestHelpdesk = new HashMap<>();
 
 
     @Given("Login helpdesk con utente test {string}")
     public void loginHelpdeskConUtenteTest(String nameFile) {
         this.datiTestHelpdesk = this.dataPopulation.readDataPopulation(nameFile+".yaml");
-        helpdeskPage.changePage();
+        String variabileAmbiente = System.getProperty("environment");
+        switch (variabileAmbiente) {
+            case "dev" -> helpdeskPage.changePage(this.datiTestHelpdesk.get("url").toString());
+            case "test", "uat" ->
+                    helpdeskPage.changePage(this.datiTestHelpdesk.get("url").toString().replace("dev", variabileAmbiente));
+            default -> Assert.fail("Non stato possibile trovare l'ambiente inserito, Inserisci in -Denvironment test o dev o uat");
```

Comment:
@leleOFA correggi il log con "Non è stato.."

Reply:
do it